.github/pull_request_template.md

```diff
@@ -50,7 +48,7 @@ We typically require tests for all PRs in order to:
 If tests are not included in your PR, please explain why (for example, are they covered by existing tests)?
 -->
 
-# Are there any user-facing changes?
+### Are there any user-facing changes?
```

Comment:
Another piece of feedback:

1. The `breaking-change` label didn't exist. I decided to add it as `Breaking Change` just now, since there isn't a good reason to kebab-case AFAIK.
2. I don't think non-committers can apply labels, right? So perhaps the instructions simply need to have a checkbox `[ ] This PR contains breaking changes` or something like that?

Reply:
Actually, I'll handle this in #33660